index.d.ts

```diff
@@ -1,7 +1,8 @@
 declare class MethodRegistry {
-    constructor(opts: object)
+  constructor(opts: Record<string, unknown>)
 
-    lookup(bytes: string): void
-    parse(signature: string): void
+  lookup(bytes: string): void
+
+  parse(signature: string): void
 }
-export = MethodRegistry
+export = MethodRegistry;
```

Comment:
Comment: You're going to get rid of this file in the TypeScript migration since it will be generated automatically, but you can make use it when you refactor.